pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -407,12 +407,11 @@ func (l *LibvirtDomainManager) asyncMigrate(vmi *v1.VirtualMachineInstance, opti
 
 		isBlockMigration := (vmi.Status.MigrationMethod == v1.BlockMigration)
 		migrationPortsRange := migrationproxy.GetMigrationPortsList(isBlockMigration)
-		loopbackAddress := network.GetLoopbackAddress()
 
 		// Create a tcp server for each direct connection proxy
 		for _, port := range migrationPortsRange {
 			key := migrationproxy.ConstructProxyKey(string(vmi.UID), port)
-			migrationProxy := migrationproxy.NewTargetProxy(loopbackAddress, port, nil, nil, migrationproxy.SourceUnixFile(l.virtShareDir, key))
+			migrationProxy := migrationproxy.NewTargetProxy("127.0.0.1", port, nil, nil, migrationproxy.SourceUnixFile(l.virtShareDir, key))
```

Comment:
@oshoval can you please explain how it works for ipv6?

Reply:
i believe that since ipv4 isnt disabled, the 127.0.0.1 and 0.0.0.0 can act as local listening
for all ports, both for ipv6 and ipv4

Reply:
https://serverfault.com/questions/256811/how-to-completely-dynamically-disable-ipv4-stack-from-linux
see last remark  for example please

Reply:
https://github.com/nodejs/node/issues/9390
0.0.0.0 works as long as bindv6only is false

127.0.0.1 works if /etc/hosts have it pointing to 127.0.0.1
https://eklitzke.org/localhost

Reply:
I'm very confused by this answer.

> [nodejs/node#9390](https://github.com/nodejs/node/issues/9390)
> 0.0.0.0 works as long as bindv6only is false
> cat /proc/sys/net/ipv6/bindv6only
> 
> 127.0.0.1 works if /etc/hosts have it pointing to localhost
> https://eklitzke.org/localhost

Correct me if I'm wrong, but this means that for it (migration) to work, we depend on a certain `/etc/hosts` configuration. That is not good, imho. What are the advantages of this approach ? 

> 
> both virt-launcher and virt-handler have it 0 and with localhost pointed by 127.0.0.1
> in kind ipv6.
> 
> in virt-launcher we would always have KUBERNETES_SERVICE_HOST
> because its a mandatory var for k8s,
> so we can use this instead of MY_POD_IP in case we want to mux between 127.0.0.1 and ::1 for cases where localhost wont have this entry, or enforce that it will have it
> wdyt ? if it can be done on different PR it would be awesome, since i really prefer to wrap this one. flakiness happens a lot and make it longer each time, another reason is that for example for the new ipv6 provider we are dual stack, we dont directly support bindv6only. sometimes coming fresher to a task make it goes smoother, and we learn from each task more.
> (for now i removed the code that looked on MY_POD_IP in the virt-launcher because it wasnt right)

Reply:
i was wrong here
the summary that was been groomed with Alona and Vladik was that as ovirt was doing, 
we are going to use ::1 / :: unless ipv6 is disabled

please see the 3rd commit, as you already did from what you told me